client/src/modules/templates/bhRenderOption.tmpl.html

```diff
@@ -0,0 +1,14 @@
+<div id="orientation" class="form-group">
+  <label class="control-label" translate>FORM.LABELS.PRINT_ORIENTATION</label>
+  <div>
+    <label class="radio-inline">
+      <input type="radio" name="portrait" id="portrait" ng-true-value='portrait' value="portrait" ng-model="$ctrl.orientation" required>
```

Comment:
According to the [AngularJS documentation](https://docs.angularjs.org/api/ng/input/input%5Bradio%5D), `ngTrueValue` is only defined for `input[type="checkbox"]`.  You should remove it from these `input[type="radio"]` buttons.